pkg/patterns/declarative/pkg/applier/direct.go

```diff
@@ -128,16 +128,19 @@ func (d *DirectApplier) Apply(ctx context.Context, opt ApplierOptions) error {
 	for i, arg := range opt.ExtraArgs {
 		switch arg {
 		case "--force":
+			// TODO Does this do anything? It seems like opt (aka ApplierOptions) is not used anymore
```

Comment:
Good question.  I _think_ it's right?  Here in this loop we map from ExtraArgs (which are very oriented towards the command line execution which we're trying to stop using) to explicit flags like `Force`.  Then below we map `applyOpts.ForceConflicts = opt.Force`.  And then we pass applyOpts to `Run`.  But I might be missing something?

We do have some fancy tests that can record the HTTP calls, so we could actually test the resulting behaviour now, but I _think_ it's fine?

Reply:
Totally correct. I missed the line where we map `opt.Force` to  `applyOpts.ForceConflicts`. So this makes sense. One could just refactor and set `applyOpts.ForceConflicts` to `true` in this line I guess.